app/models/export/submission_entry_row.rb

```diff
@@ -21,5 +16,18 @@ def values_for_additional
         value_for("Additional#{n}", default: nil)
       end
     end
+
+    private
+
+    def source_field_for(destination_field)
+      sheet_definition = sheet_definition(model._framework_short_name, model.entry_type)
+      sheet_definition.export_mappings[destination_field]
+    end
+
+    def sheet_definition(framework_short_name, entry_type)
+      "Framework::Definition::#{framework_short_name}::#{entry_type.titleize}".constantize
```

Comment:
Could this now make use of `Framework::Definition[framework_short_name]` somehow to simplify things?

Reply:
I looked at that but unfortunately it doesn't really give us anything; we have to take the constant it gives us and turn it back into a string. I thought the repetition was on balance better, although it would put the `NameError` in one place. I'll take another look tomorrow.

Reply:
Hm I've gone with what's on [the card](https://trello.com/c/i05471TJ/415-submission-entries-have-an-explicit-total-value-attribute-column) here – trying to keep the new column name and this lined up